### PR TITLE
Add configuration for verification-store and email in base_stencil

### PR DESCRIPTION
**Pull request by kmrinal19**

### CODE REVIEW

**Review by ayushjn20 — APPROVED** [review]

**Comment by rish9101** [comment]
Nice catch! 👍

**Comment by nisarg73** [comment]
@kmrinal19 Please add `emails` as well in this, and update the PR. You may refer to https://github.com/IMGIITRoorkee/omniport-docs/commit/45df2ce19c54c29dd4c14b8b747ab53d740c4061 for this.

**Comment by algomaster99** [comment]
@kmrinal19 I agree with @nisarg73 , please add the missing configurations in this PR itself.

File: configuration/base_stencil.yml [hunk 1]
```diff
@@ -40,6 +43,13 @@ services:
 integrations: {}
   # Various useful applications that can connect to Django
   # Freely drop any integration you do not use or need
+emails:
+  emailBackend: # The backend used for sending emails
+  emailHost: # The email service provider used
+  emailUseTls: # Whether an email uses a TLS connection or not
+  emailPort: # The port used by smtp server
```
Comment (algomaster99) [inline configuration/base_stencil.yml:50]: `smtp` -> `SMTP`

File: configuration/base_stencil.yml [hunk 2]
```diff
@@ -40,6 +43,13 @@ services:
 integrations: {}
   # Various useful applications that can connect to Django
   # Freely drop any integration you do not use or need
+emails:
+  emailBackend: # The backend used for sending emails
+  emailHost: # The email service provider used
+  emailUseTls: # Whether an email uses a TLS connection or not
```
Comment (algomaster99) [inline configuration/base_stencil.yml:49]: Whether an email should be encrypted via TLS or not

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Few small changes and we're good to go :smile:

**Review by algomaster99 — APPROVED** [review]
@kmrinal19 nice work 😌

**Pull request merged**